pkg/plugin/v3/scaffolds/internal/templates/dockerfile.go

```
@@ -62,7 +62,7 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager
 FROM gcr.io/distroless/static:nonroot
 WORKDIR /
 COPY --from=builder /workspace/manager .
-USER nonroot:nonroot
+USER 65532:65532
```

```suggestion
USER 1001:1001
```
Could we use this ID?